src/theme/DocVersionBanner/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react'
+import clsx from 'clsx'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+import Link from '@docusaurus/Link'
+import Translate from '@docusaurus/Translate'
+import { useActivePlugin, useDocVersionSuggestions } from '@docusaurus/plugin-content-docs/client'
```

Comment:
I think this code should be upgraded to docusaurus v3:

Locally I get:

<img width="1282" alt="image" src="https://github.com/user-attachments/assets/9924e04c-be0c-4e4b-a40e-cbfcd6a7afe2" />

Note that I rebased this PR to `main` and other PR are working; https://github.com/fastify/website/pull/264